examples/demo_mapviewer_features.ipynb

```diff
@@ -114,6 +114,45 @@
     "# center the map on a specific substation\n",
     "network_map2.center_on_substation('P1')"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Example 3\n",
+    "# creates a network that contains lines and hvdc lines, and adds a substation and a tie line.\n",
+    "# Then, adds some geographical data extensions, not present in the original network (fictitious coordinates).\n",
+    "# Finally, display the network using the mapviewer widget\n",
+    "\n",
+    "four_substations_network = pn.create_four_substations_node_breaker_network()\n",
```

Comment:
Is it possible to find another name for the network? We add a fifth substation so "four_substations_network" may be confusing.